.github/workflows/deploy-apis-to-production.yml

```diff
@@ -1194,7 +1194,7 @@ jobs:
           gcloud secrets versions access latest --secret="prod-env-view-message-broker" > message_broker.env
           kubectl create configmap --dry-run=client -o yaml --from-env-file=message_broker.env prod-view-message-broker-config | kubectl replace -f - -n production
           gcloud secrets versions access latest --secret="prod-key-view-api-google-credentials" > google_application_credentials.json
-          kubectl create configmap --dry-run=client -o yaml --from-file=google_application_credentials.json view-api-config-files | kubectl replace -f - -n production
+          kubectl create configmap --dry-run=client -o yaml --from-file=google_application_credentials.json view-api-config-files | kubectl apply -f - -n production
```

Comment:
Using apply since replace might not work in scenarios where a config map does not exist. cc @123MwanjeMike

Reply:
Hi @NoahNsimbe 
The configmap is supposed to be created the first time manually.
Using apply instead of replace to update the configmap is not advisable as it can create redundancies and other avoidable challenges.

I see the PR was already merged but I hope it can be reverted cc @Baalmart

Reply:
I can raise a PR to make the change from `apply` to `replace` instead of reverting.

Reply:
Thanks @NoahNsimbe 
That works too